app/actions/topics.js

```diff
@@ -1,10 +1,11 @@
 // Including es6-promise so isomorphic fetch will work
-import { polyfill } from 'es6-promise';
-import fetch from 'isomorphic-fetch';
+// import { polyfill } from 'es6-promise';
+// import fetch from 'isomorphic-fetch';
+import request from 'axios';
```

Comment:
No gripes with using `axios` as our library, as it looks to have better browser support and is well maintained. However, we might still have to use a promise polyfill for browsers an environment that does not support es6-promises. Found that [here](https://github.com/mzabriskie/axios#promises). 

Apart from that I'm happy for you to remove the commented lines.